apps/condo/domains/analytics/tasks/helpers/sendDailyStatistics.spec.js

```diff
@@ -0,0 +1,652 @@
+        allowedIncidentClassifiers, notAllowedIncidentClassifiers
+
+    setFakeClientMode(index)
+    jest.setTimeout(10000)
+
+
+    beforeAll(async () => {
+        setFeatureFlag(SEND_DAILY_STATISTICS_TASK, true)
+
+        admin = await makeLoggedInAdminClient()
+        statuses = await TicketStatus.getAll(admin, {})
+
+        allowedIncidentClassifiers = await IncidentClassifier.getAll(admin, {
+            category: {
+                id: INCIDENT_CATEGORY_IDS.WATER,
+            },
+            problem: {
+                id_in: [
+                    INCIDENT_PROBLEM_IDS.NO_HOT_AND_COLD_WATER,
+                    INCIDENT_PROBLEM_IDS.NOT_HOT_WATER,
+                    INCIDENT_PROBLEM_IDS.NO_COLD_WATER,
+                ],
+            },
+        })
+        notAllowedIncidentClassifiers = await IncidentClassifier.getAll(admin, {
+            id_not_in: allowedIncidentClassifiers.map(item => item.id),
+        })
+    })
+
+    describe('class UserDailyStatistics', () => {
+        describe('should return correct statistic data for user', () => {
+            let userData = {
+                client: null,
+                organizationWithAccess1: {
+                    data: null,
+                    property: null,
+                },
+                organizationWithAccess2: {
+                    data: null,
+                    property: null,
+                },
+                organizationWithoutAccess: {
+                    data: null,
+                    property: null,
+                },
+            }
+            let justEmployeeClient
+
+            beforeEach(async () => {
+                setFeatureFlag(SEND_DAILY_STATISTICS_ORGANIZATIONS_ENABLED, false)
+
+                const administratorClient = await makeEmployeeUserClientWithAbilities({
+                    canManageOrganization: true,
+                    canManageTickets: true,
+                    canReadTickets: true,
+                    canManageIncidents: true,
+                    canReadIncidents: true,
+                })
+                userData.client = administratorClient
+                userData.organizationWithAccess1.data = administratorClient.organization
+                userData.organizationWithAccess1.property = administratorClient.property
+
+                const client2 = await makeEmployeeUserClientWithAbilities({
+                    canManageOrganization: true,
+                    canManageTickets: true,
+                    canReadTickets: true,
+                    canManageIncidents: true,
+                    canReadIncidents: true,
+                })
+                await createTestOrganizationEmployee(admin, client2.organization, administratorClient.user, client2.role)
+                userData.organizationWithAccess2.data = client2.organization
+                userData.organizationWithAccess2.property = client2.property
+
+                const client3 = await makeEmployeeUserClientWithAbilities({
```

Comment:
really best to call it makeEmployeeClientWithPermissions, it is not superman 🔢 it does not have abilities :-)

Reply:
Not in this pr) Let him be superman for a little while longer))